packages/dds/tree/src/simple-tree/jsonSchema.ts

```diff
@@ -0,0 +1,197 @@
+/*!
```

Comment:
For reviewers: interested in opinions on the naming used here.

Reply:
I think how you've done it is consistent from an English language perspective, though it takes me a bit to figure out exactly how the words should be put together (it's something like "Simple" + NodeType + "Node" + "Schema" + "Kind", where some of the terms are optional). I also like this though:

"NodeJsonSchemaBase" -> "JsonSchemaNodeBase"
"ObjectNodeJsonSchema" -> "JsonSchemaObjectNode"
"JsonLeafSchemaType" -> "JsonSchemaLeafType"

and 

"SimpleNodeSchemaKind" -> "SimpleSchemaNodeKind"
"SimpleObjectNodeSchema" -> "SimpleSchemaObjectNode"

etc.

I think that it's useful to have the words "Simple" and "Schema" always together (same for JsonSchema) to help with search/lookup. "SimpleSchema" is the domain and the thing after it resides in the domain. Maybe what I really want is namespaces for these things (JsonSchema.NodeBase, JsonSchema.LeafType, JsonSchema.ObjectNode).